src/__tests__/overscroll-color-context-test.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {OverscrollColorProvider} from '../overscroll-color-context';
+import {render} from '@testing-library/react';
+import ThemeContextProvider from '../theme-context-provider';
+import {makeTheme} from './test-utils';
+import * as platform from '../utils/platform';
+
+const resizeWindow = (x: number, y: number) => {
+    window.innerWidth = x;
+    window.innerHeight = y;
+    window.dispatchEvent(new Event('resize'));
+};
+
+const DummyComponent = ({callback}: {callback: jest.Mock}) => {
+    React.useEffect(() => {
+        callback();
+    }, [callback]);
+
+    return <span>dummy component</span>;
+};
+
+test('overscroll does not cause a remount of its children', async () => {
+    resizeWindow(640, 480);
```

Comment:
if you're not changing this, how are you trying to replicate the issue in this tests? Is this test failing without your changes?

Reply:
yep, it is failing without my changes, this issue happens without the need of change window size, but in a small window size (mobile or tablet)